conda/gromacswrapper/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "gromacswrapper" %}
+{% set version = "0.8.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/GromacsWrapper-{{ version }}.tar.gz
+  sha256: 688b9708dfc5ad118db0a2fb647cae5501d285e212e07a929e8cd2177f7042e3
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - matplotlib-base
+    - numkit
```

Comment:
This package is not available yet.
I think you can submit to https://github.com/conda-forge/staged-recipes instead? We do not need to maintain every package by ourselves.

Reply:
> 

I agree with you.  
We can submit the recipes to `conda-forge`. But I found some PRs have been in progressing for a long time before it can be merged into `conda-forge`, so I also submitted numkit recipe, which can solve the problem currently.  @felix5572 Maybe would you like to join this?

Reply:
However, I don't think it needs to be summited to DP-GEN's repository.

Reply:
@njzjz We decide to publish `0.10.4` version tomorrow, and I haven't submitted the  recipes to `conda-forge` yet.  But I  still hope to merge this PR first to let offline package be available for the community. What's your opinion ?

Reply:
You also need to publish packages to our channel